src/Evaluators/common.jl

```diff
@@ -8,6 +8,12 @@ struct AutoDiffFactory <: AbstractAutoDiffFactory
     ∇f::AdjointStackObjective
 end
 
+function transfer!(target::AutoDiffFactory, origin::AutoDiffFactory)
```

Comment:
How about overloading `Base.copy!` there?

Reply:
Same as below... it's not really a copy. it assumes both were initialized with the same case.